agents/thorlabs_mc2000b/chopper_mc2000b_agent.py

```diff
@@ -0,0 +1,339 @@
+import time
+import os
+
+from ocs import ocs_agent, site_config
+from ocs.ocs_twisted import TimeoutLock
+import argparse
+import txaio
+
+ON_RTD = os.environ.get("READTHEDOCS") == "True"
+if not ON_RTD:
+    from MC2000B_COMMAND_LIB import *  # noqa: F403
+    os.add_dll_directory("C:\\Program Files (x86)\\Thorlabs\\MC2000B\\Sample\\Thorlabs_MC2000B_PythonSDK")
+
+# For logging
+txaio.use_twisted()
+LOG = txaio.make_logger()
+
+bladetype_keys = {'MC1F2': 0,
+                  'MC1F10': 1,
+                  'MC1F15': 2,
+                  'MC1F30': 3,
+                  'MC1F60': 4,
+                  'MC1F100': 5,
+                  'MC1F10HP': 6,
+                  'MC1F2P10': 7,
+                  'MC1F6P10': 8,
+                  'MC1F10A': 9,
+                  'MC2F330': 10,
+                  'MC2F47': 11,
+                  'MC2F57B': 12,
+                  'MC2F860': 13,
+                  'MC2F5360': 14}
+
+outputmode_keys = {'target': 0,
+                   'actual': 1}
+
+reference_mode_keys = {'internal': 0,
+                       'external': 1}
+
+reference_high_prec_mode = {'internalouter': 0,
+                            'internalinner': 1,
+                            'externalouter': 2,
+                            'externalinner': 3}
+
+
+class ControllerAgent:
```

Comment:
"Controller" is perhaps a bit overloaded of a word to use in "ControllerAgent". Other instances are "Motion Controller" in the fts aerotech agent, "PID Controller" in the HWP rotation agent, "Turbo Controller" in the Pfeiffer TC400 Agent, "Pysmurf Controller" in the pysmurf controller agent.

Along with the rename from "chopper_mc2000b" can we replace all instances of "ControllerAgent" with "ThorlabsMC2000BAgent"? This is here, where the class object is instantiated, in the call to `site_config.parse_args()`, and in the docs config example.